libs/web/components/src/ProposalNewForm.tsx

```diff
@@ -125,3 +127,21 @@ export const ProposalNewForm: FC<
 		</form>
 	);
 };
+
+const useMarkdownValidation = (value: string) => {
```

Comment:
Don't think you need to do this, as the browser has native `maxlength` validation that does the same thing I believe. Can you try and switch to it?

Reply:
nice it actually stops typing at that length too